scripts/release.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
This script will correctly update the version/version.go file and create a tag for us after which we'll promote to a release.